templates/CommentsInterface_singlecomment.ss

```diff
@@ -34,7 +34,11 @@
 				<% end_if %>
 			</div>
 			<% if $RepliesEnabled %>
-				<a class="comment-reply-link" href="#{$ReplyForm.FormName}">Reply to $AuthorName.XML</a>
+                <% if $JavaScriptEnabled %>
+				    <a class="comment-reply-link" href="#{$ReplyForm.FormName}">Reply to $AuthorName.XML</a>
+                <% else %>
+                    <a class="comment-reply-link" href="$Parent.Link?replyTo=$ID#Form_ReplyForm_$ID">Reply to $AuthorName.XML</a>
```

Comment:
Reducing the above to the followings works with the include_js tag toggled for true and false.

```
<% if $RepliesEnabled %>
 <a class="comment-reply-link" href="$Parent.Link?replyTo=$ID#Form_ReplyForm_$ID">Reply to $AuthorName.XML</a>
<% end_if %>
```